datahub-web-react/src/app/search/SearchBar.tsx

```diff
@@ -149,6 +142,7 @@ export const SearchBar = ({
     hideRecommendations,
     showQuickFilters,
     viewsEnabled = false,
+    combineSiblings = false,
```

Comment:
so we default want to not combine siblings in our search bar? is there somewhere where we particularly don't want them combined?

Reply:
Right the idea was to make siblings an opt-in behavior, where the home/searchResults pages have that behavior but not assume that all use cases in the UI would get that behavior was my thought.